preshed/bloom.pyx

```diff
@@ -109,8 +209,8 @@ cdef void bloom_add(BloomStruct* bloom, key_t item) nogil:
     cdef key_t one = 1 # We want this explicitly typed, because bits
     hash128_x86(&item, sizeof(key_t), 0, &keys)
     for hiter in range(bloom.hcount):
-        hv = (keys[0] + (hiter * keys[1])) % bloom.length
-        bloom.bitfield[hv // sizeof(key_t)] |= one << (hv % sizeof(key_t))
+        hv = (keys[0] + (hiter * keys[1])) % bloom.length # length is in BITS
```

Comment:
I wonder if we should rename `BloomStruct.length` to something like `BloomStruct.bits_length`.

I think that normally it doesn't make too much sense to put units in variable names (better to have them in types, but Cython), but I think it's really easy to confuse the units here.

Anyway, this could be a separate PR if we want this.

Reply:
I hope that won't be ambiguous going forward, but changing the name like that is definitely a good way to avoid further issues. I won't add it to this PR for now though.